### PR TITLE
PAAS-1998: handle EC deployment deletion when deleting a jahia

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
environment having augmented search

JIRA issue: https://jira.jahia.org/browse/PAAS-1998

Short description:

### CODE REVIEW

File: mixins/elasticsearch.yml [hunk 1]
```diff
@@ -331,22 +331,28 @@ actions:
             creds_bytes = (new java.lang.String(auth_basic_es_credentials)).getBytes("UTF-8");
             encoded_auth_basic_creds = Base64.getEncoder().encodeToString(creds_bytes);
 
+            nodes = jelastic.env.control.GetNodeGroups("${env.envName}", session)
+            jahia_env = (nodes.object.filter(function (object) {return object.name == "proc"}).length == 1);
+
+            indices_prefix = jahia_env ? "${env.shortdomain}__as_*" : "${env.shortdomain}__jc-*"
```
Comment (kevinvandel) [inline mixins/elasticsearch.yml:337]: Really ? The prefix ends with an underscore for AS but a dash for jCustomer ? :grimacing:

Comment (Ronan-Gorain) [reply mixins/elasticsearch.yml:337]: yes :/

File: packages/jahia/update-events.yml [hunk 2]
```diff
@@ -339,6 +340,15 @@ onBeforeDelete:
           - if (${response.out} == 1):
               refreshUnomiAllowedIPs:
                 unomiEnvName: ${globals.unomi_env_name}
+
+  - script: |-
+      const augsearch = jelastic.env.control.GetNodeGroups("${env.envName}", session).object.filter(function (object) {
+          return object.name == "cp";
+      }).pop().augsearch;
+      return {'result': 0, 'removeECDeployement': (augsearch != null)}
```
Comment (kevinvandel) [inline packages/jahia/update-events.yml:348]: I think we can put this in a mixin action, it's most likely we'll need it again in the future. Like `isAugsearchEnabled` (in jahia mixin)

Comment (Ronan-Gorain) [reply packages/jahia/update-events.yml]: Done

**Review by kevinvandel — APPROVED** [review]

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**